openage/util/fslike/path.py

```diff
@@ -63,6 +64,8 @@ def __init__(self, fsobj, parts=None):
 
         self.fsobj = fsobj
 
+        self.is_temp = is_temp
```

Comment:
This should be set to `False` by default

Reply:
![image](https://github.com/SFTtech/openage/assets/96500764/26ced371-ab37-4337-82de-4da0d97c7c61)